CMakeLists.txt

```diff
@@ -67,6 +67,7 @@ set(GINKGO_CUDA_COMPILER_FLAGS "" CACHE STRING
 set(GINKGO_CUDA_ARCHITECTURES "Auto" CACHE STRING
     "A list of target NVIDIA GPU achitectures. See README.md for more detail.")
 option(GINKGO_CUDA_DEFAULT_HOST_COMPILER "Tell Ginkgo to not automatically set the CUDA host compiler" OFF)
+option(GINKGO_HIP_AMD_UNSAFE_ATOMIC "Compiler uses unsafe floating point atomic (only for AMD GPU and ROCM >= 5). Default is ON because we use hipMalloc, which is always on coarse grain. Must turn off when allocating memory on fine grain" ON)
```

Comment:
We are lacking an explanation what coarse grain and fine grain allocations are somewhere in the documentation. Are those the terms AMD uses somewhere? Maybe link to that

Reply:
So basically hipMalloc is fine, only hipMallocHost causes issues? We don't use it, so that's fine :) I think the documentation should mainly refer to the consequences of this choice.

Reply:
yes, I add the ref in the comment